examples/pipeline/dummy_entity_linking.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+"""Demonstrate how to build a simple knowledge base and run an Entity Linking algorithm.
+Currently still a bit of a dummy algorithm: taking simply the entity with highest probability for a given alias
+"""
```

Comment:
Also nitpicky, but we should add `from __future__ import unicode_literals` here and all `.py` files, just in case and for consistency.